ci-operator/step-registry/mirror-images/by-oc-adm/mirror-images-by-oc-adm-ref.yaml

```diff
@@ -12,5 +12,10 @@ ref:
   - namespace: test-credentials
     name: openshift-custom-mirror-registry
     mount_path: /var/run/vault/mirror-registry
+  dependencies:
+  - name: "release:latest"
+    env: OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE
+  - name: "release:latest"
+    env: RELEASE_IMAGE_LATEST
```

Comment:
@lwan-wanglin I think this env can be removed

Reply:
@yunjiang29  Checked with @aleskandro in private before, we think can keep it compliant with the rest of the codebase. there are many place set the both, do you recommend to remove it?

Reply:
I'm just not sure it is ok, there is to be a reason for which the installer jobs use it (at least a historical reason).

We can try removing it though.